beetsplug/fetchart.py

```diff
@@ -979,7 +979,7 @@ def commands(self):
         cmd.parser.add_option(
             u'-q', u'--quiet', dest='quiet',
             action='store_true', default=False,
-            help=u'shows only quiet art'
+            help=u'only output about art that is fetched, or cannot be found'
```

Comment:
Maybe `u'quiet mode: do not output albums that already have artwork'` is even clearer?